lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -118,6 +119,9 @@ class TwilioConnectionTransport extends StateMachine {
           options.realm,
           eventPublisherOptions)
       },
+      _automaticSubscription: {
```

Comment:
@makarandp0 Nitpick: Can you move this above `_dominantSpeaker` and below `_accessToken`?